src/pyscaffoldext/markdown/extension.py

```diff
@@ -38,13 +41,19 @@ def add_long_desc(content: str) -> str:
     updater = ConfigUpdater()
     updater.read_string(content)
     metadata = updater["metadata"]
-    metadata["long-description"].value = "file: README.md"
-    long_desc_type = "long-description-content-type"
+
+    long_desc = metadata.get(dasherize(DESC_KEY)) or metadata.get(DESC_KEY)
+    long_desc.name = DESC_KEY  # dash-separated keys are deprecated in setuptools
```

Comment:
ConfigUpdater do have a setter for the `name` property but this does not seem to be working to change the generated string...

Reply:
Sorry for that, I understood it wrong. `name` is for Section objects... Option objects operate with `key`.